epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -34,7 +34,7 @@ public void bind(@SuppressWarnings("rawtypes") EpoxyModel model,
 
     if (previouslyBoundModel != null) {
       // noinspection unchecked
-      model.bind(objectToBind(), previouslyBoundModel);
+      model.bindChanges(objectToBind(), previouslyBoundModel);
```

Comment:
Fixed it this morning already. Like I said, it's unfinished, but it's still good that you point it out, otherwise I might have missed it.